modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -41,6 +42,110 @@
+
+=== Align to new accounts.google.com authorization URI
+
+Core 10.12.1 brought an update of the google/apiclient from version 2.12.6 to 2.13.1. However, in version 2.13.0 the accounts.google.com authorization URI has been updated. This change broke old code that uses the "setApprovalPrompt('force')" instead of the newer "setPrompt('consent')" method, as this endpoint does not support the legacy approval prompt parameter. This has been now fixed. https://github.com/owncloud/core/pull/40783[#40783]
+
+=== Fix query used to delete thumbnails
+
+Fixed query that detects unused thumbnails to prevent unnecessary deletes and potential recreations. https://github.com/owncloud/core/pull/40801[#40801]
+
+=== Automatically disable online updater for enterprise
+
+Online updater is not recommended for Enterprise installations and is now automatically disabled in such cases. https://github.com/owncloud/core/pull/40841[#40841]
+
+=== Versions expire job does not error anymore with federated shares
+
+The versions expire job does not error anymore with federated shares when versioning metadata is enabled. https://github.com/owncloud/core/pull/40847[#40847]
+
+
+=== Notable Changes
+
+* Setup checks now allow other values other than "none" for X-Robots-Tag header. If "none" or "noindex" and "nofollow" are missing, a security warning is raised. Previously a header value with "noindex" and "nofollow" wasn't allowed even though it was valid. https://github.com/owncloud/core/pull/40715[#40715]
+* The Last Login column on the Users page is now correctly hidden if the setting is initially unchecked. https://github.com/owncloud/core/pull/40771[#40771]
+* User input is validated now in UserController. https://github.com/owncloud/core/pull/40769[#40769]
+* A system tag with a too long name resulted in potentially inaccessible data. https://github.com/owncloud/core/pull/40804[#40804]
+* A federated share with a too long name resulted in inaccessible data. https://github.com/owncloud/core/pull/40726[#40726]
+* Due to Apache rewrite rules originally not existing headers could hold an empty string. https://github.com/owncloud/core/pull/40856[#40856]
+* Verbose command output of "background:queue:execute" is now displayed. https://github.com/owncloud/core/pull/40844[#40844]
+* There have been rare cases where deleting a file from a Google Drive mount could have thrown an undefined variable error. Though the process completed without further issues, no errors should have been thrown. This fix initializes the variables for these cases properly making the error go away. https://github.com/owncloud/core/pull/40822[#40822]
```

Comment:
```suggestion
* There have been rare cases where deleting a file from a Google Drive mount could have thrown an undefined variable error. Since the process completed without further issues, no errors should have been thrown. This fix initializes the variables for these cases properly, making the error go away. https://github.com/owncloud/core/pull/40822[#40822]
```